### PR TITLE
add notifme-template typings

### PR DESCRIPTION
**Pull request by ziv**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project. **created from module written in Babel using it's Flow output**
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
Just as a heads up, your commits don't seem to be associated with your GitHub account. While this isn't technically a problem, you might care if you want more appropriate attribution. You can either make sure your GitHub account is associated with the email address you're using for your commits, or rebase and amend your commits to fix the author name and email.

If you don't care about any of this stuff, I can always just pull in the changes.

File: types/notifme-template/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
+// Type definitions for notifme-template 1.0
+// Project: https://github.com/notifme/notifme-template
+// Definitions by: Ziv <https://github.com/ziv>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/
+// TypeScript Version: 3.2
+
+/// <reference types="node" />
+
+export default function RenderFactory(renderer: RendererType, folder: string, options?: OptionsType): RendererFunction;
```
Comment (uniqueiniquity) [inline types/notifme-template/index.d.ts]: It seems like the [underlying library](https://github.com/notifme/notifme-template/blob/master/index.js) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

**Review by ziv — COMMENTED** [review]
seems right

File: types/notifme-template/index.d.ts (hunk 1 above)
Comment (uniqueiniquity) [reply types/notifme-template/index.d.ts]: I'm not sure why you marked this as resolved; is there something I'm missing about why this .d.ts should be using `export default`? Please let me know if there's any part I should explain more.

**Pull request closed without merging**